.devcontainer/Dockerfile

```diff
@@ -15,7 +15,10 @@
 # For details, see https://github.com/devcontainers/images/tree/main/src/base-ubuntu
 FROM mcr.microsoft.com/devcontainers/base:1-ubuntu-22.04
 
+COPY .go-version /tmp/.go-version
+
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && export VERSION=$(cat /tmp/.go-version) \
```

Comment:
This variable doesn't seem consumed

Reply:
The VERSION variable is consumed within the devcontainer, and this can be observed during local testing.

Reply:
Doesn't seem so.

The variable scope is limited to this RUN instuction

Reply:
You're right, it used the latest version. 😭

Reply:
Yes, currently, it can only be specified explicitly. Attempts to automate the setup go version using environment variables or other methods have not been successful.